pkg/kubelet/lifecycle/handlers.go

```diff
@@ -55,9 +58,16 @@ func (hr *HandlerRunner) Run(containerID kubecontainer.ContainerID, pod *api.Pod
 		var buffer bytes.Buffer
 		output := ioutils.WriteCloserWrapper(&buffer)
 		err := hr.commandRunner.ExecInContainer(containerID, handler.Exec.Command, nil, output, output, false)
+		if err != nil {
+			glog.V(1).Infof("Lifecycle-Probe for Container %q in Pod %q failed - %q", container.Name, pod.Name, buffer.String())
```

Comment:
nits addressed.

On Fri, May 20, 2016 at 4:01 PM, Yu-Ju Hong <notifications@github.com>
wrote:

> In pkg/kubelet/lifecycle/handlers.go
> <https://github.com/kubernetes/kubernetes/pull/25681#discussion_r64118703>
> :
>
> > @@ -55,9 +58,16 @@ func (hr *HandlerRunner) Run(containerID kubecontainer.ContainerID, pod *api.Pod
> >  		var buffer bytes.Buffer
> >  		output := ioutils.WriteCloserWrapper(&buffer)
> >  		err := hr.commandRunner.ExecInContainer(containerID, handler.Exec.Command, nil, output, output, false)
> > +		if err != nil {
> > +			glog.V(1).Infof("Lifecycle-Probe for Container %q in Pod %q failed - %q", container.Name, pod.Name, buffer.String())
>
> nit: use the pkg/kubelet/util/format util to print the pod information:
> s/pod.Name/format.Pod(pod)
>
> nit2: how about replacing "Probe" with "hook" and print the handler or
> command, so that it's clear which hook failed.
>
> Same for the http handler below.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/25681/files/b7721421008b473c7dd227ec3dab52a15e099332#r64118703>
>